docs/AboutUs.md

```diff
@@ -21,29 +21,27 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Jane Doe
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/janedoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
 
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Jonathan Tan
 
```

Comment:
Is there a need to do About Us in this PR?

Reply:
yep was an accident haha